Userland/Applications/Browser/Tab.cpp

```diff
@@ -446,6 +449,33 @@ Tab::Tab(Type type)
     add_search_engine("Google", "https://google.com/search?q={}");
     add_search_engine("Yandex", "https://yandex.com/search/?text={}");
 
+    auto custom_search_engine = GUI::Action::create_checkable("Custom", [&](auto& action) {
+        String search_engine;
+        if (GUI::InputBox::show(window(), search_engine, "Enter Format:", "Custom Search Engine") != GUI::InputBox::ExecOK || search_engine.is_empty()) {
```

Comment:
```suggestion
        if (GUI::InputBox::show(window(), search_engine, "Enter format:", "Custom Search Engine") != GUI::InputBox::ExecOK || search_engine.is_empty()) {
```

Shouldn't capitalize every word.

Reply:
Fixed